RELEASE_NOTES.rst

```diff
@@ -175,8 +175,7 @@ are added for existing bears.
 
 New bears:
 -  ``AutoPrefixBear`` (Add vendor prefixes automatically in CSS)
--  ``ClangComplexityBear`` (Calculates cyclomatic complexity of each function
```

Comment:
https://github.com/coala/coala-bears/issues/919 has been created about this problem.  Please mention it in your commit message.